browser/src/layer/tile/CalcTileLayer.js

```diff
@@ -515,6 +518,7 @@ L.CalcTileLayer = L.CanvasTileLayer.extend({
 
 		var scrollSection = app.sectionContainer.getSectionWithName(L.CSections.Scroll.name);
 		scrollSection.stepByStepScrolling = true;
+		this._syncTileContainerSize();
```

Comment:
Status is sent very often in calc, mostly it doesn't change the sheet (where I belive issue appears).
Can we add some guard here to not touch sizing too much?

Reply:
example: someone else added new sheet, will it force redraw our view? or similar case

Reply:
It turns out that we don't need the call there.